src/screens/Home/BalanceSection.js

```diff
@@ -35,37 +37,36 @@ import { formatFiatValue, formatFiatChangeExtended } from 'utils/format';
 import { useThemeColors } from 'utils/themes';
 import { spacing } from 'utils/variables';
 
-// Types
-import type { Balance } from 'models/Home';
-
 // Local
 import SpecialButton from './components/SpecialButton';
 
 
 type Props = {|
-  balance: Balance,
+  balanceInFiat: BigNumber,
+  changeInFiat?: ?BigNumber,
```

Comment:
I assume it can either be provided `maybe` BigNumber or not provided at all? So either `changeInFiat: ?BigNumber` or `changeInFiat?: BigNumber`, wdyt?

```suggestion
  changeInFiat: ?BigNumber,
```

Reply:
`prop?: BigNumber` - means that the prop is optional and can be BigNumber or undefined
`prop: ?BigNumber` - means that the prop must be present & can be BigNumber or null or undefined.
`prop?: ?BigNumber` - means that the prop is option and can be BigNumber or null or undefined.

The third case is what we want to support atm, so we need both of these `?`.